.storybook/preview.cjs

```diff
@@ -6,4 +10,12 @@ export const parameters = {
       date: /Date$/,
     },
   },
-}
\ No newline at end of file
+};
+
+export const decorators = [
+  (Story) => {
+    return React.createElement(ThemeProvider, {
+      children: React.createElement(Story),
+    });
+  },
```

Comment:
```suggestion
  (Story) => (
    <ThemeProvider>
      <Story />
    </ThemeProvider>
  ),
```

Reply:
I would love to but I've got this error from storybook:
```
[vite] Internal server error: Failed to parse source for import analysis because the content contains invalid JS syntax. You may need to install appropriate plugins to handle the .cjs file format, or if it's an asset, add "**/*.cjs" to `assetsInclude` in your configuration.
  Plugin: vite:import-analysis
  File: /Users/kzuraw/Developer/macaw-ui/.storybook/preview.cjs:19:21
  17 |      <ThemeProvider>
  18 |        <Story />
  19 |      </ThemeProvider>
     |                      ^
  20 |    ),
  21 |  ];
```